src/tokens/_spacing.scss

```diff
@@ -0,0 +1,92 @@
+$ant: (
+  clamp: (
+    min: 0.8rem,
+    target: 1vw,
```

Comment:
I'm a bit wary about using `vw` here as I'm getting flashbacks of Bits' shrinking container as you get to a bigger screen size. The max clamp though should keep that in check I guess?